src/main/java/com/kilopo/kosshop/config/DatabaseConfig.java

```diff
@@ -16,17 +17,16 @@
 import javax.sql.DataSource;
 import java.util.Properties;
 
+import static com.kilopo.kosshop.constants.Constants.BASE_PACKAGE;
+import static com.kilopo.kosshop.constants.Constants.Database.PROPERTIES;
 import static org.hibernate.cfg.Environment.DIALECT;
 import static org.hibernate.cfg.Environment.SHOW_SQL;
 import static org.hibernate.cfg.Environment.HBM2DDL_AUTO;
 
-import static constants.Constants.Database.PROPERTIES;
-import static constants.Constants.KOSSHOP;
-
 @Configuration
 @EnableTransactionManagement
 @PropertySource(PROPERTIES)
-@ComponentScan(KOSSHOP)
+@ComponentScan(BASE_PACKAGE)
```

Comment:
use constants like:

@PropertySource(Constants.Database.PROPERTY_SOURCE)
@ComponentScan(basePackages = {Constants.BASE_PACKAGE})

Because name "PROPERTIES" ambiguous.